web/satellite/src/components/onboardingTour/steps/OverviewStep.vue

```diff
@@ -105,6 +108,12 @@ import { PM_ACTIONS } from '@/utils/constants/actionNames';
 export default class OverviewStep extends Vue {
     public isLoading: boolean = false;
 
+    private readonly analytics: AnalyticsHttpApi = new AnalyticsHttpApi();
+          
+    public onRcloneClick(): void {
+        this.analytics.linkEventTriggered(AnalyticsEvent.PATH_SELECTED, "Rclone Sync");
+        }  
```

Comment:
this line should have one less indentation

Reply:
done